de.json

```
@@ -706,11 +706,11 @@
         "CameraControl.Settings.AimInFrontOfHead" : "Ausrichtung auf die Stirn",
         "CameraControl.Settings.HideCamera" : "Kamera unsichtbar",
         "CameraControl.Settings.ForceEyesOnCamera" : "Fokussiere Augen auf Kamera",
-        "CameraControl.Settings.HideBadges" : "Abzeichen unsichtbar",
-        "CameraControl.Settings.HideLasers" : "Laser unsichtbar",
+        "CameraControl.Settings.HideBadges" : "Namen ausblenden",
```

This one is about badges instead of names :p
```suggestion
        "CameraControl.Settings.HideBadges" : "Abzeichen ausblenden",
```

---

This only hides the badges, not the entire nameplate?
If so, agreed.
Assumed it would also hide the nameplate in which case either would be interchangeable.

---

I updated it, thanks for the heads-up @Avunia